src/azul/lambda_layer.py

```diff
@@ -132,6 +131,12 @@ def _generate_requirements(self):
                 f.write(req.to_pin() + '\n')
 
     def _build_package(self, input_zip: Path, output_zip: Path):
+        # Delete Chalice's build cache because our layer cache eviction rules
+        # are stricter and we want a full rebuild.
+        try:
```

Comment:
ux nit: There should be a logger.debug statement in this method. When debugging the layer generation, it's frustrating that things are not verbose. Doing so enables us to look at the logs during `make deploy` and find statement of things that occurred, rather than try to replicate what has happened at a later time in order to try to understand the state of the deployment. 

Another place would be the removal of the vendored packages from the layer requirements, it's not obvious that it's occurring unless you dig into the code.